### PR TITLE
Added type definition for electron-notify and mock-require

### PR DESCRIPTION
**Pull request by djpereira**
case 1. Add a new type definition.
- [X] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [X] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

File: electron-notify/electron-notify.d.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// Type definitions for electron-notify v0.1.0
+// Project: https://github.com/hankbao/electron-notify
+// Definitions by: Daniel Pereira <https://github.com/djpereira>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../github-electron/github-electron.d.ts" />
+
+declare namespace ElectronNotify {
+
+  interface ICloseNotificationEvent {
```
Comment (paulvanbrenk) [inline electron-notify/electron-notify.d.ts]: Can you stick the interfaces in the 'electron-notify' module, and export them so they can be used?

**Comment by paulvanbrenk** [comment]
Looks good, other than the comment I had

**Comment by djpereira** [comment]
@paulvanbrenk, I moved the interfaces as requested.
I am still getting familiar with the conventions. 
That's why I initially thought that it was a good idea to keep the interfaces within the namespace to avoid conflicts, as they still could have been used via `ElectronNotify.IMyInterface`, and just export what's really exported in the original module.
Thanks.

**Comment by djpereira** [comment]
@paulvanbrenk, is this good for merging now? Let me know if you have any other comments.

**Pull request merged**